src/VisualStudio/Core/Def/Implementation/PullMemberUp/MainDialog/PullMemberUpDialog.xaml

```diff
@@ -0,0 +1,245 @@
+﻿<vs:DialogWindow
```

Comment:
would love if we could pull out the xaml/dialog stuff from this review. it's making it much more complex and time consuming.